packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx

```diff
@@ -1,15 +1,14 @@
-import { Button, ButtonGroup } from "@components/button";
-import { Content, Footer } from "@components/placeholders";
+import { Content } from "@components/placeholders";
 import { Field, Label } from "@components/field";
 
 import { Div } from "@components/html";
 import { Form } from "@components/form";
-import { Heading } from "@components/typography";
 import { Image } from "@components/image";
 import { Launch } from "./assets";
 import { Popover } from "@components/popover";
 import { TextInput } from "@components/text-input";
```

Comment:
why not the text input?

Reply:
I could try replacing this one. I kept wondering if we only wanted to replace ones that were slots or not.